hydrogym/core.py

```diff
@@ -317,20 +360,25 @@ def step(self, iter: int, control: Iterable[float] = None, **kwargs):
 
   def reset(self):
     """Reset variables for the timestepper"""
-    pass
+    self.t = 0.0
```

Comment:
See above - time should be owned by the flow, and the reset method for the flow can also reset the time.